docs/_macros/classes.md

```diff
@@ -66,6 +66,10 @@ PUBLIC
 ANNOTATION
 <td>PATTERN</td>
 <td>The class must have an annotation that matches the pattern. The set of annotations is all annotations in the class, also the annotations on fields and methods.</td>
+
+META_ANNOTATION
```

Comment:
Is META_ANNOTATION really the right name? Because this will report direct annotations as well as meta annotations. That is, it is a super set of ANNOTATION.

Reply:
suggestion?

Reply:
We settled on `INDIRECTLY_ANNOTATED`.

Reply:
ÜBER_ANNOTATION, obviously.